src/Transaction/BaseFirestoreTransactionRepository.ts

```diff
@@ -31,7 +31,7 @@ export class TransactionRepository<T extends IEntity> extends AbstractFirestoreR
   }
 
   execute(queries: IFireOrmQueryLine[]): Promise<T[]> {
-    const query = queries.reduce((acc, cur) => {
+    const query = queries.reduce((acc: CollectionReference | Query, cur) => {
```

Comment:
I'm pretty sure that the accumulator can only be type Query

Reply:
I found weird too, but after activating the strict mode the TS compiler complained about the signature of this method. I believe the problem is that the initial value for the reducer is `this.firestoreColRef` (type is `CollectionReference`) but the function returns `acc.where(cur.prop, op, cur.val)` (type is `Query`). Does it make sense? Do you see a better solution?

Reply:
Oh, you're right! 

I see two possible solutions:

1- `this.firestoreColRef as Query` ugly but... it works?
2- Remove the reduce altogether and make it a forEach, I'd require mutating a copy of `this.firestoreColRef` instead of the reference

Don't know which one is the most "correct way" of doing it though

Reply:
I believe the second path is the "correct" one since we do not cast any types, even though we lose immutability

Reply:
I noticed that the `Transaction` folder was not included in the configuration, so this change was not needed (I found this error while playing around with random files and forgot to remove the fix). I will make the changes discussed above at a later PR